packages/talisman-ui/src/components/MysticalBackgroundV3/MysticalBackgroundV3.tsx

```diff
@@ -1,111 +1,110 @@
-import { classNames } from "@talismn/util"
-import { CSSProperties, FC, memo, useEffect, useMemo, useRef, useState } from "react"
+import { CSSProperties, memo, useLayoutEffect, useMemo, useRef } from "react"
 import { useMeasure } from "react-use"
 
 import { MYSTICAL_PHYSICS_V3, MysticalPhysicsV3 } from "./MysticalPhysicsV3"
-import { ParentSize, useCelestialArtifact } from "./useCelestialArtifact"
-
-const CelestialArtifact = memo(
-  ({
-    parentSize,
-    config,
-    color,
-  }: {
-    parentSize: ParentSize
-    config: MysticalPhysicsV3
-    // force color
-    color?: string
-  }) => {
-    const [id] = useState(() => crypto.randomUUID())
-    const artifact = useCelestialArtifact(config, parentSize, color)
-
-    const refInitialized = useRef(false)
-
-    useEffect(() => {
-      refInitialized.current = false
-    }, [])
-
-    const [style1, style2] = useMemo<[CSSProperties, CSSProperties]>(
-      () => [
-        {
-          stopColor: artifact.color,
-          stopOpacity: 0.9,
-          transitionProperty: "stop-color",
```

Comment:
The primary source of our flickering was the repainting required to transition the `stop-color` on these gradients.